controllers/admin/AdminEmailsManager.php

```diff
@@ -49,11 +49,19 @@ public function initContent()
                 foreach ($settings['inputs'] as $input) {
                     $value = Tools::getValue($input['name']);
                     if ($current['name'] == $template && isset($current['inputs'][$input['name']])) {
-                        $this->context->smarty->assign($input['name'], $current['inputs'][$input['name']]);
+                        if (isset($input['lang']) && $input['lang'] == true) {
+                                $this->context->smarty->assign($input['name'], $current['inputs'][$input['name']][Context::getContext()->language->id]);
```

Comment:
I think that you should check if the value is set for this id_lang before accessing it.